### ISSUE TITLE
q-option-group component states that v-model is required when actually it's the value prop

### ISSUE THREAD

**Issue by jesuisse** [quasarframework/quasar-framework.org#314]
### Software version

Quasar: v0.14.7

### What did you get as the error?

The http://quasar-framework.org/components/option-group.html documentation states that q-option-group needs a `v-model`. This doesn't work; vue complains about a missing prop `value` on the q-option-group component.

When you replace replace the `v-model ` with `:value`, everything works fine. I suggest someone update the documentation to reflect the change in the code if that's expected behaviour.

### What were you expecting?

That `v-model ` would work as advised in the documentation.

### What steps did you take, to get the error?

Here's a snippet of code that doesn't work (with display-flags correctly set as an array in data):

    <q-option-group
       type="toggle"
          v-model='display_flags'
          :options="[ {label: 'Something', value: 'first'},
                      {label: 'Something else', value: 'second'}]"/>

Doesn't work, vue complains about a missing prop `value`. Replace `v-model` with `:value` and everything works.

**Comment by rstoenescu** [comment]
Vue-compiler aliases "v-model" to `:value="val" @input="new_val => val = new_val"` https://vuejs.org/v2/guide/components.html#Form-Input-Components-using-Custom-Events

So "v-model required" is same as "value prop required".

Check display_flags variable as when component is first rendered it is not initialised hence the error.

Due to the fact that the variable is an array in this case, setting `value` to it means the reference to the array is passed in. Changing an array index within the Quasar component makes it actually work with just setting "value" (due to the explanation above) without the `@input` listener, BUT this approach violates the intended way of using v-model. It may work in this case (because arrays are references to a place in memory) but will fail in many others, so it's not a recommended approach.

**Bottom line**, use v-model and check that display_flags actually contains something when the QOptionGroup is actually rendered for the first time.
